src/cdk/testing/tests/cross-environment.spec.ts

```diff
@@ -246,6 +246,15 @@ export function crossEnvironmentSpecs(
       const subcomps = await harness.directAncestorSelectorSubcomponent();
       expect(subcomps.length).toBe(2);
     });
+
+    it('should handle a compound selector with an ancestor', async () => {
```

Comment:
can we add a test for a selector with some quoted content to test that code path?